plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1203,7 +1203,7 @@ def onNextSampleButton(self):
             image_id = sample["id"]
             image_file = sample.get("path")
             image_name = sample.get("name", image_id)
-            node_name = sample.get("PatientID", sample.get("name", image_id))[-20:]
```

Comment:
have u checked when patience name is too long..

Reply:
Yes. The elided node name appears in slice view, but the dropdown text still appears full length, or at least longer than that in the slice view.
#841 was supposed to elide the dropdown text, but it seems not to have worked. I don't know Qt so I can't comment further.